serverless-map-reduce/amr.js

```diff
@@ -0,0 +1,61 @@
+const invoke = (() => {
```

Comment:
I should have pointed it out in the original PR, but we prefer the simpler syntax for function declarations

```JavaScript
function invoke() {
...
}
```

On that note, I highly reccomend adding a linter to this example. Here is a package.json with a valid lint command

https://github.com/binaris/functions-examples/blob/bedfe0c4d154a915da5c3065aa19a5cc626f7564/fractal-terrain-generation/package.json

A . file `.eslintrc.js` is also required


```
module.exports = {
  "extends": "airbnb-base",
  "plugins": [
    "import"
  ],
  "rules": {
    "no-underscore-dangle": "off",
    "no-restricted-syntax": "off",
    "no-unused-vars": ["error", {
      argsIgnorePattern: "^unused",
    }],
  },
  "parserOptions": {
    "sourceType": "script"
  }
};
```